specutils/analysis/flux.py

```diff
@@ -116,3 +118,57 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None):
     ew =  dx - (line_flux / continuum)
 
     return ew.to(calc_spectrum.spectral_axis.unit)
+
+def is_continuum_near_zero(spectrum, eps=0.01):
+    """
+    Determine if the baseline continuum of the spectrum is near zero.
+
+    The value is scaled to match the sigma/standard deviation parameter of a
```

Comment:
Is this paragraph maybe copied from something else?  I don't really see how it applies to this function

Reply:
Yup... will remove.